### PR TITLE
Ignore comment in .jvmopts on Windows

### PR DESCRIPTION
**Pull request by exoego**
I think this PR may fix https://github.com/sbt/sbt/issues/3908

### CODE REVIEW

**Comment by exoego** [comment]
It looks like PR https://github.com/sbt/sbt-launcher-package/pull/210 fixes appveyor issue.

**Comment by dwijnand** [comment]
Trying to switch to a shared AppVeyor account, and get it to re-run. Closing and re-opening.

File: citest/test3/test3.bat [hunk 1]
```diff
@@ -0,0 +1,11 @@
+@echo on
+
+SETLOCAL
+
+SET JAVA_HOME=C:\jdk9
+SET PATH=C:\jdk9\bin;%PATH%
+SET SBT_OPTS=-Xmx4g -Dfile.encoding=UTF8
+
+"..\freshly-baked\sbt\bin\sbt" check
```
Comment (dwijnand) [inline citest/test3/test3.bat]: This is failing on AppVeyor with:

File: src/universal/bin/sbt.bat [hunk 2]
```diff
@@ -35,9 +35,10 @@ IF DEFINED JAVA_HOME SET "PATH=%JAVA_HOME%\bin;%PATH%"
 
 rem users can set JAVA_OPTS via .jvmopts (sbt-extras style)
 IF EXIST .jvmopts FOR /F %%A IN (.jvmopts) DO (
-  SET JAVA_OPTS=%%A !JAVA_OPTS!
```
Comment (exoego) [inline src/universal/bin/sbt.bat]: Test failure. Need investigation

```
C:\projects\sbt-launcher-package\citest\test3>"..\freshly-baked\sbt\bin\sbt" check 
Error: Could not find or load main class #-Duser.timezone=GMT
Caused by: java.lang.ClassNotFoundException: #-Duser.timezone=GMT
The filename, directory name, or volume label syntax is incorrect.
Error: Could not find or load main class #-Duser.timezone=GMT
Caused by: java.lang.ClassNotFoundException: #-Duser.timezone=GMT
Error: Could not find or load main class #-Duser.timezone=GMT
Caused by: java.lang.ClassNotFoundException: #-Duser.timezone=GMT
Command exited with code 1
```

**Pull request merged**